packages/web-components/src/components/style-picker/utilities/renderSvg.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { svg } from 'lit';
+
+/**
+ *
+ * @param {object} svgData - object from @carbon/pictograms package
+ */
+export const renderSvg = (svgData) => {
+  // Create SVG element using svg template tag
+  return svg`
+      <svg 
+        xmlns=${svgData.attrs.xmlns}
```

Comment:
minor, but i think deconstructing these would look better
```
const { content, attrs } = svgData;
<svg { ...attrs }>{content.map ... }</svg>
```
also you may want to include `...rest` for additional properties.